sections/05_treat.md

```diff
@@ -156,7 +156,7 @@ repositioning via deep learning is currently missing.
 
 #### Ligand-based prediction of bioactivity
 
-In the biomedical domain, high-throughput chemical screening aims to improve
+High-throughput chemical screening aims to improve
```

Comment:
I found the previous phrasing awkward and moves the true subject away from the topic position, but I see your point. Changed to "screening in biomedical research" in michaelmhoffman/deep-review@c82e777e96597381100ab0551a24e27bfdfc36c6.